pkg/bindings/connection.go

```diff
@@ -149,23 +152,83 @@ func sshClient(_url *url.URL, uri string, identity string, machine bool) (Connec
 	connection := Connection{
 		URI: _url,
 	}
+	userinfo := _url.User
+	if _url.User == nil {
+		u, err := user.Current()
+		if err != nil {
+			return connection, fmt.Errorf("current user could not be determined: %w", err)
+		}
+		userinfo = url.User(u.Username)
+	}
 	port := 22
 	if _url.Port() != "" {
 		port, err = strconv.Atoi(_url.Port())
 		if err != nil {
 			return connection, err
 		}
 	}
+	// ssh_config
+	alias := _url.Hostname()
+	match := false
+	cfg := ssh_config.DefaultUserSettings
+	if val := cfg.Get(alias, "User"); val != "" {
+		userinfo = url.User(val)
+		match = true
+	}
+	if val := cfg.Get(alias, "Hostname"); val != "" {
+		uri = val
+		match = true
+	}
+	if val := cfg.Get(alias, "Port"); val != "" {
+		if val != ssh_config.Default("Port") {
+			port, err = strconv.Atoi(val)
+			if err != nil {
+				return connection, fmt.Errorf("port is not an int: %s: %w", val, err)
+			}
+			match = true
+		}
+	}
+	if val := cfg.Get(alias, "IdentityFile"); val != "" {
+		if val != ssh_config.Default("IdentityFile") {
+			identity = strings.Trim(val, "\"")
+			match = true
+		}
+	}
+	if match {
+		logrus.Debugf("Alias: %s", alias)
+		logrus.Debugf("  User: %s", userinfo)
+		logrus.Debugf("  Hostname: %s", uri)
+		logrus.Debugf("  Port: %d", port)
+		logrus.Debugf("  IdentityFile: %s", identity)
+	}
 	conn, err := ssh.Dial(&ssh.ConnectionDialOptions{
 		Host:                        uri,
 		Identity:                    identity,
-		User:                        _url.User,
+		User:                        userinfo,
 		Port:                        port,
 		InsecureIsMachineConnection: machine,
 	}, ssh.GolangMode)
 	if err != nil {
 		return connection, newConnectError(err)
 	}
+	// socket path
+	if _url.Path == "" {
+		session, err := conn.NewSession()
+		if err != nil {
+			return connection, err
+		}
+		defer session.Close()
+
+		var b bytes.Buffer
+		session.Stdout = &b
+		if err := session.Run(
+			"podman info --format '{{.Host.RemoteSocket.Path}}'"); err != nil {
+			return connection, err
+		}
+		val := strings.TrimSuffix(b.String(), "\n")
+		_url.Path = val
+		logrus.Debugf("Path: %s", val)
```

Comment:
`Remote Socket Path: %s`?

Reply:
It was Path as in url.Path, but I think it can be just removed... There is nothing printed for url.User or url.Port